workbench-core/datasets/src/dataSetService.ts

```diff
@@ -50,16 +52,33 @@ export class DataSetService {
    * @returns the DataSet object which is stored in the backing datastore.
    */
   public async provisionDataSet(request: CreateProvisionDatasetRequest): Promise<DataSet> {
-    const { storageProvider, ...dataSet } = request;
-
-    await storageProvider.createStorage(dataSet.storageName, dataSet.path);
-
-    const provisioned: DataSet = {
-      ...dataSet,
-      storageType: storageProvider.getStorageType()
+    const metadata: Metadata = {
+      actor: { id: '-', roles: [] },
```

Comment:
is `-` a placeholder value?

Reply:
In this case, as we will need those for creating the access permissions.
There is another spot where we aren't and for now, won't be passed the authenticated user however someone made `actor` a required field.

Reply:
oh wait, it's optional! I'll just remove it until I'm ready for it.